draft-ietf-keytrans-protocol.md

```diff
@@ -1005,71 +1244,120 @@ that the result equals the root value of the prefix tree.
 
 ## Combined Tree {#proof-combined-tree}
```

Comment:
I found this section quite hard to parse, and I don't think I've fully understood it. It started clicking when I arrived at line 1486. You try to define a general data format to accommodate for multiple types of proofs.

Would it maybe help to first list the use cases (i.e., what is included) and then generalize that into one data structure? The use-cases don't explicitly refer to the fields in the `CombinedTreeProof` data structure anyways.